internal/config/config.go

```diff
@@ -76,7 +76,7 @@ func (c *Config) GetOrResolveCABundle() []byte {
 func (c *Config) initialize() {
 	c.initializeEnvs()
 	c.initializeCert()
-	c.initializeCABundle()
+	// c.initializeCABundle()
```

Comment:
We should not be commenting this. Please set the caBundle env in the deployment spec and set it to the ca value generated in the helm chart.